test/E2ETests/TestAssets/SuiteLifeCycleTestProject/SuiteLifeCycleTestClass_AssemblyInitializeAndCleanup.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+#if NET6_0_OR_GREATER
+using System.Threading.Tasks; 
+#endif
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SuiteLifeCycleTestProject;
+
+[TestClass]
+public class SuiteLifeCycleTestClass_AssemblyInitializeAndCleanup : IDisposable
+#if NET6_0_OR_GREATER
+        , IAsyncDisposable 
+#endif
+{
+    private static TestContext s_testContext;
+
+    public TestContext TestContext { get; set; }
+
+    public SuiteLifeCycleTestClass_AssemblyInitializeAndCleanup()
+    {
+        s_testContext.WriteLine("Ctor was called");
+    }
+
+    [AssemblyInitialize]
+    public static void AssemblyInit(TestContext testContext)
```

Comment:
I was expecting to have a class with only the assembly init/cleanup methods so that it's clearer they are run separately. I am not sure if you have to have the class marked with `TestClass` when you only have assembly init/cleanup.